<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Find</class>
 <widget class="QDialog" name="Find">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>96</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>214</width>
    <height>96</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="radioButton_Point">
     <property name="text">
      <string>点</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="radioButton_Segment">
     <property name="text">
      <string>段</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="radioButton_AGV">
     <property name="text">
      <string>AGV</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>请输入要查询的Id号：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_id"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Find</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Find</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
